docs/guides/request_loaders.mdx

```diff
@@ -39,20 +39,31 @@ classDiagram
 
 class BaseStorage {
     <<abstract>>
-    _attributes_
-    _methods_()
+    + id
+    + name
+    + open()
+    + drop()
 }
 
 class RequestLoader {
     <<abstract>>
     _attributes_
-    _methods_()
+    + fetch_next_request()
+    + mark_request_as_handled()
+    + is_empty()
+    + is_finished()
+    + get_handled_count()
+    + get_total_count()
+    + to_tandem()
 }
 
 class RequestManager {
     <<abstract>>
     _attributes_
```

Comment:
Are the `_attributes_` lines necessary?

Reply:
Otherwise there is an empty row. But they are not necessary. What do you think?

![image](https://github.com/user-attachments/assets/73bbafef-2c86-4cf3-b963-ad94fd1a403b)

Reply:
Nice, but unfortunately, it seems to work only for entirely empty classes.

![image](https://github.com/user-attachments/assets/5839b30f-28e9-458e-a9b0-927c8db158b1)

Additionally, there doesn't seem to be any further configuration options available at this moment - [Mermaid Class Diagram Config Properties](https://mermaid.js.org/config/schema-docs/config-defs-class-diagram-config.html#classdiagramconfig-properties).

So I would probably leave it as it is for now.

Reply:
It may be worth it to raise that with the maintainers... In my opinion, the empty attributes section is marginally better than the "attributes" label.

Reply:
Okay, updated.